src/query/service/src/databases/system/build_options_table.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use common_catalog::table::Table;
+use common_catalog::table_context::TableContext;
+use common_exception::Result;
+use common_expression::types::BooleanType;
+use common_expression::types::StringType;
+use common_expression::utils::FromData;
+use common_expression::DataBlock;
+use common_expression::TableDataType;
+use common_expression::TableField;
+use common_expression::TableSchemaRefExt;
+use common_meta_app::schema::TableIdent;
+use common_meta_app::schema::TableInfo;
+use common_meta_app::schema::TableMeta;
+use common_storages_system::SyncOneBlockSystemTable;
+use common_storages_system::SyncSystemTable;
+
+build_info::build_info!(pub fn build_info);
```

Comment:
Since build info only checks the current crate, it looks like we'll have to put this file into service.